src/chia_plot.cpp

```diff
@@ -16,6 +16,17 @@
 #include <chrono>
 #include <iostream>
 
+#ifdef __linux__ 
+	#include <unistd.h>
+	#define GETPID getpid
+#elif _WIN32
+	#include <processthreadsapi.h>
+	#define GETPID GetCurrentProcessId
+#else
+	#define GETPID
```

Comment:
that won't compile, how about: `#define GETPID() int(-1)`

Reply:
My apologies, I haven't tested it so I had no idea about that.